downstream/modules/hub/proc-pull-image.adoc

```diff
@@ -4,16 +4,16 @@
 
 
 [role="_abstract"]
-You can pull images from the {HubName} container registry to make a copy to your local machine. 
+You can pull images from the {HubName} container registry to make a copy to your local machine.
 
 .Prerequisites
 
 * You must have permission to view and pull from a private container repository.
 
 .Procedure
 
-. If are pulling container images from a password or token-protected registry, xref:proc-create-credential[create a credential in {ControllerName}] before pulling the image. 
-. From the navigation panel, select menu:Execution Environments[Execution Environments].
+. If are pulling container images from a password or token-protected registry, xref:proc-create-credential[create a credential in {ControllerName}] before pulling the image.
```

Comment:
The attribute is good but there's a "you" missing between "if" and "are" :)

Reply:
Fixed.